package.json

```diff
@@ -28,5 +28,10 @@
   },
   "prettier": {
     "singleQuote": true
+  },
+  "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^6.3.0",
+    "@fortawesome/free-solid-svg-icons": "^6.3.0",
+    "@fortawesome/react-fontawesome": "^0.2.0"
```

Comment:
These should be in the client package not the root one

Reply:
Alright, go ahead and take a look! Thanks Amauri!